tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordSession.cs

```diff
@@ -178,5 +223,63 @@ public async Task Sanitize(IEnumerable<RecordedTestSanitizer> sanitizers, bool s
                 }
             }
         }
+
+        /// <summary>
+        /// Creates a copy of the provided record entry.
+        /// Used primarily for sanitization logging.
+        /// </summary>
+        /// <param name="requestEntry">The record entry to copy.</param>
+        /// <returns>The copied record entry.</returns>
+        private RecordEntry CopyRecordEntry(RecordEntry requestEntry)
```

Comment:
This should exist in the `RecordEntry` class. Call it `Clone()` IMO.

Reply:
I didn't do that since I am not actually cloning the whole object, just the pieces I need for the comparisons.
Now that you asked, I'll do it anyway.